sidebars.js

```diff
@@ -269,9 +269,93 @@ module.exports = {
         }
       ]
     },
-    { type: 'doc', id: 'reference/core-concepts' },
-    { type: 'doc', id: 'community' },
-    { type: 'doc', id: 'contributing/contributing' },
-    { type: 'doc', id: 'changelog' }
+    {
+      type: 'category',
+      label: 'Reference',
+      collapsed: true,
+      items: [
+        {
+          type: 'category',
+          label: 'Glossary of Expectations',
+          items: [
+            { type: 'doc', id: 'reference/core-concepts' }
+          ]
+        },
+        {
+          type: 'category',
+          label: 'Core Concepts',
+          items: [
+            { type: 'doc', id: 'reference/core-concepts' },
+            { type: 'doc', id: 'reference/checkpoints-and-actions' },
+            { type: 'doc', id: 'reference/data-context' },
+            { type: 'doc', id: 'reference/data-discovery' },
+            { type: 'doc', id: 'reference/data-docs' },
+            { type: 'doc', id: 'reference/datasources' },
+            { type: 'doc', id: 'reference/evaluation-parameters' },
+            { type: 'doc', id: 'reference/expectations/conditional-expectations' },
+            { type: 'doc', id: 'reference/expectations/distributional-expectations' },
+            { type: 'doc', id: 'reference/expectations/expectations' },
+            { type: 'doc', id: 'reference/expectations/implemented-expectations' },
+            { type: 'doc', id: 'reference/expectation-suite-operations' },
+            { type: 'doc', id: 'reference/execution-engine' },
+            { type: 'doc', id: 'reference/metrics' },
+            { type: 'doc', id: 'reference/profilers' },
+            { type: 'doc', id: 'reference/expectations/result-format' },
+            { type: 'doc', id: 'reference/expectations/standard-arguments' },
+            { type: 'doc', id: 'reference/stores' },
+            { type: 'doc', id: 'reference/dividing-data-assets-into-batches' },
+            { type: 'doc', id: 'reference/validation' }
```

Comment:
If we order these in a thoughtful way (rather than alphabetically), it would be easier to accept these changes.  Currently they feel like a pretty big departure from how the corresponding content was organized in RTD. Would be happy to discuss further

Reply:
Will handle in GDOC-142